elementary/monitor/dbt_project/models/alerts/alerts.sql

```diff
@@ -33,7 +33,7 @@ select
     *,
     false as alert_sent,  {# backwards compatibility #}
```

Comment:
I don't think this is needed.
If we'd remove it, it'll default to `NULL` which is okay.

(Same for all alert tables)

Reply:
The problem is that because we have backwards compatibility, we check the content of the column so we could set the relevant suppression status for old alerts.
So if we'll remove the column, new users won't have it in their tables and we will get an SQL error for trying to read an not existing column.

So if we need to keep this column for backwards compatibility, lets keep the alert_sent with relevant data instead of nulls 🙂